VideoWeb/VideoWeb/ClientApp/src/app/testing/mocks/data/conference-test-data.ts

```diff
@@ -174,9 +174,9 @@ export class ConferenceTestData {
             scheduled_duration: 45,
             status: ConferenceStatus.NotStarted,
             participants: participants,
-            judge_i_frame_uri: 'judgeiframe@kinly..com',
-            participant_uri: 'participant@kinly..com',
-            pexip_node_uri: 'node@kinly..com',
+            judge_i_frame_uri: 'https://test.hearings.com/judge',
```

Comment:
Indeed. I had a look at the console and it was throwing errors because it could not recognise the input. I've ammended this and also verified the judge iframe source property is set when method that uses this property is called